6-random-numbers/Random Numbers/Random Numbers.cpp

```diff
@@ -0,0 +1,30 @@
+#include <iostream>
+#include <time.h>
+using namespace std;
+
+int main()
+{
+    srand((unsigned)time(0)); 
+    int r, inValue;
+	float numbers[21] = {};
```

Comment:
I think that it's not nice to have an array that's bigger than the amount of different random numbers. An array with 20 numbers should have been enough.